README.md

```diff
@@ -30,7 +30,7 @@ On every machine in the cluster:
 
 2. Install prerequisites: 
    
-   **Ubuntu 16.04, 16.10, 17.04, Windows Subsystem for Linux (WSL)**
+   **Debian/Ubuntu**
```

Comment:
I'm removing WSL from the README file. WSL now supports various distributions including opensuse and fedora. WSL users should use their distribution's package manager to install the dependencies.